.yarnrc.yml

```diff
@@ -1,5 +1,7 @@
 initScope: hashicorp
 
+nodeLinker: node-modules
```

Comment:
This configures yarn 2 to use `/node_modules` in addition to its `.yarn/cache/*`